server/src/modules/mentorship-request/mentorship-request.service.ts

```diff
@@ -12,7 +12,17 @@ export class MentorshipRequestService {
   ): Promise<void> {
     const user = await User.findAll();
     mentorshipRequest.menteeId = user[0].id;
+    mentorshipRequest.requestStatus = RequestStatus.PENDING;
     // TODO: Replace with the actual logged in menteeId
     await MentorshipRequest.create(mentorshipRequest);
   }
+
+  async getRequestsForMentor(
+    mentorId: number,
+  ): Promise<MentorshipRequestGetDTO[] | null> {
+    return MentorshipRequest.findAll({
+      where: { mentorId, requestStatus: RequestStatus.PENDING },
```

Comment:
Ah I see: I guess `requestStatus` makes this query simpler? There might be a way to query for `where: {mentorshipResponse: null}` or something but no worries, this is probably fine.